velox/vector/arrow/Bridge.cpp

```diff
@@ -759,6 +759,23 @@ TypePtr importFromArrow(const ArrowSchema& arrowSchema) {
       }
       break;
 
+    case 'd': { // decimal types.
+      try {
+        std::string::size_type sz;
+        // Parse "d:".
+        if (format[1] == ':') {
+          int precision = std::stoi(&format[2], &sz);
+          // Parse ",".
+          if (format[2 + sz] == ',') {
+            int scale = std::stoi(&format[2 + sz + 1], &sz);
+            return DECIMAL(precision, scale);
+          }
+        }
+      } catch (std::invalid_argument) {
+        // Fall through.
```

Comment:
Why we fall through here?

Reply:
@Yuhta I thought the default VELOX_USER_FAIL should suffice. But after your comment, I realized it might hide bugs if the fall-through cases happen to match by chance.
I added a VELOX_USER_FAIL now.